Getting-Started.adoc

```diff
@@ -1,27 +1,27 @@
-= Sk8s: a FaaS for Kubernetes - Getting Started
-
-IMPORTANT: We don't have a curated and published release yet, so please use the manual build process documented in the link:README.adoc[README] for the time being.
+= Getting Started with riff
 
 == Installation
 
-=== [[helm]]Helm install of Sk8s FaaS
+=== [[helm]]Helm install of riff
 
-Sk8s offers a Helm Chart for deploying the FaaS and its required services to a Kubernetes Cluster.
-The following instructions cover how to initialize Helm and install Sk8s FaaS on the Minikube cluster.
+Riff offers a helm chart for deploying the FaaS and its required services to a Kubernetes Cluster.
+The following instructions cover how to initialize helm and install riff.
 
 ==== Prerequisites
 
-* A running 1.7 Kubernetes cluster with at least 4GB of memory. These instructions assume minikube for now.
+* A running 1.7+ Kubernetes cluster with at least 4GB of memory.
+
+For Minikube use:
 
 ----
-minikube start --cpus=4 --memory=4096 --disk-size=64g
+minikube start --cpus=4 --memory=4096
```

Comment:
I would even remove the cpus setting, that has to vary by system

Reply:
ok